{"uid":"5ddf09594efb7ea1","name":"Проверка невозможно поменять никнейм на дубликат который есть уже в система","fullName":"tests.test_editing_profile.TestEditingProfile#test_it_is_impossible_to_save_an_already_taken_nickname","historyId":"fa24c345791f29b3f0d6a917ee673e7e","time":{"start":1738874903346,"stop":1738874903346,"duration":0},"description":"Проверяется на пользователе Aleska","descriptionHtml":"<p>Проверяется на пользователе Aleska</p>\n","status":"broken","statusMessage":"KeyError: 'access_token'","statusTrace":"request = <SubRequest 'login_to_app' for <Function test_it_is_impossible_to_save_an_already_taken_nickname>>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"021d2297b6e1bad49b22ab76c2e3d84a\")>\n\n    @pytest.fixture(scope='function')\n    @allure.title('Авторизация пользователя в системе')\n    @allure.step('Авторизация пользователя в системе')\n    def login_to_app(request, driver):\n        \"\"\" Фикстура Логина в систему \"\"\"\n        login_page = ApiAuthorization(driver)\n>       login_page.user(User.LOGIN, User.PASSWORD)\n\nconftest.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.api.authorization.ApiAuthorization object at 0x7f49b7b5e840>\nusername = 'Aleska', password = 'qwe123'\n\n    def user(self, username: str, password: str) -> None:\n        \"\"\"\n        Метод авторизации пользователя по средствам API.\n    \n        :param username: Логин пользователя\n        :param password: Пароль пользователя\n        \"\"\"\n        headers = {\n            'X-Client-Id': 'round',\n            'X-Client-Type': 'web',\n            'X-Language': 'ru'\n        }\n        data = {\n            'client_id': 'Krujok.Online.Mobile',\n            'grant_type': 'password',\n            'username': username,\n            'password': password\n        }\n    \n        response = requests.post(url=Api.TOKEN, headers=headers, data=data)\n>       access_token = response.json()[\"access_token\"]\nE       KeyError: 'access_token'\n\nmodel/api/authorization.py:30: KeyError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1738874454511,"stop":1738874454512,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1738874903346,"stop":1738874903760,"duration":414},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1738874903760,"stop":1738874903761,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Инициализация локаторов","time":{"start":1738874903761,"stop":1738874903761,"duration":0},"status":"passed","steps":[{"name":"Инициализация локаторов","time":{"start":1738874903761,"stop":1738874903761,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'elements' for <Function test_it_is_impossible_to_save_an_already_taken_nickname>>"}],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},{"name":"Авторизация пользователя в системе","time":{"start":1738874903761,"stop":1738874905212,"duration":1451},"status":"broken","statusMessage":"KeyError: 'access_token'\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 53, in login_to_app\n    login_page.user(User.LOGIN, User.PASSWORD)\n  File \"/usr/workspace/model/api/authorization.py\", line 30, in user\n    access_token = response.json()[\"access_token\"]\n                   ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n","steps":[{"name":"Авторизация пользователя в системе","time":{"start":1738874903761,"stop":1738874905212,"duration":1451},"status":"broken","statusMessage":"KeyError: 'access_token'\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 53, in login_to_app\n    login_page.user(User.LOGIN, User.PASSWORD)\n  File \"/usr/workspace/model/api/authorization.py\", line 30, in user\n    access_token = response.json()[\"access_token\"]\n                   ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'login_to_app' for <Function test_it_is_impossible_to_save_an_already_taken_nickname>>"},{"name":"driver","value":"<selenium.webdriver.chrome.webdriver.WebDriver (session=\"021d2297b6e1bad49b22ab76c2e3d84a\")>"}],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"driver::0","time":{"start":1738874905219,"stop":1738874905405,"duration":186},"status":"passed","steps":[],"attachments":[{"uid":"95c9292561de9591","name":"page_source","source":"95c9292561de9591.html","type":"text/html","size":39},{"uid":"6655dbb54799ffed","name":"screenshot","source":"6655dbb54799ffed.png","type":"image/png","size":10592},{"uid":"8e7c028b0744fa92","name":"browser_logs","source":"8e7c028b0744fa92.txt","type":"text/plain","size":0}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"labels":[{"name":"severity","value":"Critical"},{"name":"feature","value":"Редактирование профиля"},{"name":"tag","value":"regression"},{"name":"tag","value":"editing_profile"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_editing_profile"},{"name":"subSuite","value":"TestEditingProfile"},{"name":"host","value":"c6a72fb339a9"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_editing_profile"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":9,"skipped":0,"passed":33,"unknown":0,"total":42},"items":[{"uid":"8888dc4befa2dafa","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1738871588675,"stop":1738871588675,"duration":0}},{"uid":"411d5cf9b548961a","status":"broken","statusDetails":"KeyError: 'access_token'","time":{"start":1738864923516,"stop":1738864923516,"duration":0}},{"uid":"656ebbf44c041e2f","status":"passed","time":{"start":1738268589816,"stop":1738268594780,"duration":4964}},{"uid":"711a799595deca17","status":"passed","time":{"start":1738099026521,"stop":1738099031467,"duration":4946}},{"uid":"605d9386f26d0a38","status":"passed","time":{"start":1738005038709,"stop":1738005043624,"duration":4915}},{"uid":"3d388bde7199a618","status":"passed","time":{"start":1738002367675,"stop":1738002373140,"duration":5465}},{"uid":"76d37517bf726cfc","status":"broken","statusDetails":"urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=41879): Read timed out. (read timeout=120)","time":{"start":1737928135344,"stop":1737928135344,"duration":0}},{"uid":"eb6219288f4d904f","status":"passed","time":{"start":1737842646818,"stop":1737842653303,"duration":6485}},{"uid":"dc3672199d05b3cf","status":"passed","time":{"start":1737831932492,"stop":1737831939519,"duration":7027}},{"uid":"3a27995e08f31497","status":"passed","time":{"start":1737799269162,"stop":1737799275657,"duration":6495}},{"uid":"e2f300930f3276fe","status":"passed","time":{"start":1737757542549,"stop":1737757549015,"duration":6466}},{"uid":"290ee831302a9c1f","status":"passed","time":{"start":1737744500192,"stop":1737744506704,"duration":6512}},{"uid":"5de3624a2d16abb","status":"passed","time":{"start":1737652313148,"stop":1737652319617,"duration":6469}},{"uid":"3219b0f6d50140a7","status":"passed","time":{"start":1737569911394,"stop":1737569918531,"duration":7137}},{"uid":"b9976257f5b9d93c","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium-browser is no longer running, so ChromeDriver is assuming that Chrome has crashed.)","time":{"start":1737568961436,"stop":1737568961436,"duration":0}},{"uid":"259ec37231858339","status":"passed","time":{"start":1734034337975,"stop":1734034343956,"duration":5981}},{"uid":"4c428c48e34fcbea","status":"passed","time":{"start":1733937017181,"stop":1733937023879,"duration":6698}},{"uid":"85830cef16269e2c","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//button[@data-testing=\"after_register_dialog__btn_choose_later\"]')","time":{"start":1733865468351,"stop":1733865468351,"duration":0}},{"uid":"ddf8a3fb189057b2","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//button[@data-testing=\"after_register_dialog__btn_choose_later\"]')","time":{"start":1733859947841,"stop":1733859947841,"duration":0}},{"uid":"816b0587c40d4bf","status":"passed","time":{"start":1733856584049,"stop":1733856588482,"duration":4433}}]},"tags":["editing_profile","regression"]},"source":"5ddf09594efb7ea1.json","parameterValues":[]}